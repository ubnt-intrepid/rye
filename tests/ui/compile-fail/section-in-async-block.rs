fn main() {}

#[rye::test]
fn section_in_async_block() {
    let _ = std::convert::identity(async {
        section!("section", {
            assert!(true);
        });
    });
}
