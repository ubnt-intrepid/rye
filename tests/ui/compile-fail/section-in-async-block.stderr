error: section cannot be described at here
 --> $DIR/section-in-async-block.rs:6:9
  |
6 | /         section!("section", {
7 | |             assert!(true);
8 | |         });
  | |__________^
